CONTRIBUTING.md

```diff
@@ -1,20 +1,37 @@
 # Contributing Guidelines for Girlscript Website Boilerplate.  🚀
 
 ![GSSOC'20](https://img.shields.io/badge/GSSOC-20-orange?style=for-the-badge)
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
 
-We value all the people who are interested in contributing to Girlscript Chennai website. This is required to create a unified development environment.
+We value all the people who are interested in contributing to Girlscript Chennai website. This is required to create a unified development environment.These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
 
 We want to make contributing to this project as easy and transparent as possible, whether it's:
 * Reporting a bug
 * Submitting a fix
 * Proposing new features
 
 ## How to contribute ☘️:
+
+### 👇 Prerequisites:
+
+- Ability to use Git and Github is mandatory
+- Working knowledge of HTML and CSS
+- Basic knowledge of ReactJS is a plus and willingness to learn more
```

Comment:
instead of react(not used at all) you can write various forms of material cards as they are mostly used in the whole website.

Reply:
Or you can write basic knowledge of JS

Reply:
okay :)